packages/styling/src/styles/theme.ts

```diff
@@ -62,6 +62,10 @@ export function loadTheme(theme: IPartialTheme): ITheme {
 export function createTheme(theme: IPartialTheme): ITheme {
   let newPalette = { ...DefaultPalette, ...theme.palette };
 
+  if (!newPalette.accent) {
```

Comment:
this will never run because .accent is already defined in default palette
what you should do is

let first = { ...DefaultPalette, ...theme.palette } // populate original palette slots
let second = _doStuff(first); // where doStuff() says first.accent = first.themePrimary;
let thirdAndFinal = { ...second, ...theme.palette } // now have custom values overwrite default ones that were populated in 'second'